thentos-core/src/Thentos/Frontend/CSRF.hs

```diff
@@ -0,0 +1,99 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE PackageImports        #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TypeOperators         #-}
+module Thentos.Frontend.CSRF where
```

Comment:
i just added an explicit export list that's probably incomplete.  (will push soon!)